### DISCUSSION TITLE
📣 GitHub Copilot Certification Course Winners  📣

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#147410] [Copilot News and Announcements]
### A round of applause for our certification voucher winners! 👏 👏 👏 👏  

Congratulations to everyone who participated in our [GitHub Copilot Certification](https://github.com/orgs/community/discussions/144443) prep course! It’s been amazing to see your energy and engagement throughout these weeks. We hope the resources and support we shared have given you the confidence to take on the certification exam with ease.

### **Certification Voucher Winners** 🏆  

Your enthusiasm blew us away, making this competition incredibly close. Originally, we planned for 10 winners, but the high level of participation made it impossible not to add one more! 🎉  

Here are the **11 champions** who will receive a **free GitHub Certification exam voucher** (in alphabetical order):  🥁 🥁 

- @Borhanitrash  
- @ButeoButeo  
- @Geoffreywiseman  
- @jccampanero  
- @johnnieng  
- @juanlujand  
- @Ronaldo-001  
- @Ruslan-intech  
- @Salgado2004  
- @selendemir  
- @Vatsal-Sharma2003  

If you see your username on the list, give yourself a well-deserved pat on the back! 🎉 Then, claim your voucher by filling out [this form](https://forms.gle/2fJ2VVdNWnaSxNs87) by EOY, December 31, 2024, 10 AM EST. Once we have your details, our team will send out the vouchers.

### What’s Next? ⏭️  

1. **Take the Exam**: Sign up for the [Copilot Certification Exam](https://examregistration.github.com/login?ReturnUrl=%2Foverview) and join the growing ranks of certified experts! ✅  
2. **Share Your Thoughts**: Head to the discussion to share your feedback on the prep course and the exam experience—whether good, bad, or anywhere in between. 💬  
3. **Stay Tuned**: We're not stopping here! More exciting opportunities to upskill and engage with the GitHub community are on the way in 2️⃣ 0️⃣ 2️⃣ 5️⃣ 🚀  

If you want to know more about the GitHub Certification Program, see the [GitHub Certification Program FAQs](https://examregistration.github.com/faq).

Thank you all for making this journey such a success, and best of luck to everyone preparing for the certification exam!

**Comment by borhanitrash** [comment]
Thank you so much for this amazing opportunity! 🙌 Being a part of this prep course was an enriching experience, and winning the voucher is truly the cherry on top. Huge congratulations to all my fellow winners—let’s ace the certification exam together! 🚀

**Comment by ruslan-intech** [comment]
Thank you very much for the most exciting and engrossing contest in 2024!

**Comment by Vatsal-Sharma2003** [comment]
Thank you @Akash1134 sir for this opportunity.

**Comment by Salgado2004** [comment]
Thank you for opportunity and congratulations for all the winners 😄🚀 Also, thank you @Akash1134 for the organization of the exam prep! The materials and contributions were extremelly useful and it was a great pleasure to participate! 
For the certification.. good luck for us! I hope everybody will come back to share great results 🏅🚀

**Reply by juanlujand** [reply]
Thank you for opportunity and congratulations for all the winners🚀!!! And thank you @Akash1134 for the organization of the exam prep and patience!
For the certification.. good luck for us , and it was a really good experience to share with you this amazing Copilot learning Journey!!

**Comment by johnnieng** [comment]
Thank you so much for this wonderful opportunity! 🌟 I'm thrilled and honored to be one of the winners. The GitHub Copilot Certification prep course was incredibly enriching, and I'm grateful for the resources and support provided.

Looking forward to taking the exam and joining the ranks of certified GitHub experts. Congratulations to all the winners and participants, and a big thank you to the GitHub team for making this possible! 🎉

**Comment by Ronaldo-001** [comment]
Thank you so much @Akash1134 @queenofcorgis  for this incredible opportunity and for recognizing our efforts throughout the prep course! 🙌 I'm truly honored to be among the winners, and the resources and support provided were invaluable in boosting my confidence for the certification exam.  Looking forward to taking the next step and continuing this amazing learning journey with GitHub! 🚀

**Comment by selendemir** [comment]
OMG!!!! I'm so happy to receive this opportunity! Thank you so much for this chance, @Akash1134  @queenofcorgis , and, of course, GitHub! I filled out the form, and I hope to get certified soon!

**Comment by ButeoButeo** [comment]
Wow!!! I’m absolutely thrilled and overjoyed to receive this incredible opportunity! Thank you so much for this chance, @Akash1134, @queenofcorgis, and, of course, GitHub! 🎉 I’ll fill out the form and can’t wait to get certified! I feel like a kid on Christmas Eve, full of excitement and anticipation! Huge thanks to @Akash1134 and @queenofcorgis for all the hard work in preparing the course, it’s been invaluable!

**Comment by vpulyala** [comment]
> ### A round of applause for our certification voucher winners! 👏 👏 👏 👏
> Congratulations to everyone who participated in our [GitHub Copilot Certification](https://github.com/orgs/community/discussions/144443) prep course! It’s been amazing to see your energy and engagement throughout these weeks. We hope the resources and support we shared have given you the confidence to take on the certification exam with ease.
> 
> ### **Certification Voucher Winners** 🏆
> Your enthusiasm blew us away, making this competition incredibly close. Originally, we planned for 10 winners, but the high level of participation made it impossible not to add one more! 🎉
> 
> Here are the **11 champions** who will receive a **free GitHub Certification exam voucher** (in alphabetical order): 🥁 🥁
> 
> * @borhanitrash
> * @ButeoButeo
> * @geoffreywiseman
> * @jccampanero
> * @johnnieng
> * @juanlujand
> * @Ronaldo-001
> * @ruslan-intech
> * @Salgado2004
> * @selendemir
> * @Vatsal-Sharma2003
> 
> If you see your username on the list, give yourself a well-deserved pat on the back! 🎉 Then, claim your voucher by filling out [this form](https://forms.gle/2fJ2VVdNWnaSxNs87) by EOY, December 31, 2024, 10 AM EST. Once we have your details, our team will send out the vouchers.
> 
> ### What’s Next? ⏭️
> 1. **Take the Exam**: Sign up for the [Copilot Certification Exam](https://examregistration.github.com/login?ReturnUrl=%2Foverview) and join the growing ranks of certified experts! ✅
> 2. **Share Your Thoughts**: Head to the discussion to share your feedback on the prep course and the exam experience—whether good, bad, or anywhere in between. 💬
> 3. **Stay Tuned**: We're not stopping here! More exciting opportunities to upskill and engage with the GitHub community are on the way in 2️⃣ 0️⃣ 2️⃣ 5️⃣ 🚀
> 
> If you want to know more about the GitHub Certification Program, see the [GitHub Certification Program FAQs](https://examregistration.github.com/faq).
> 
> Thank you all for making this journey such a success, and best of luck to everyone preparing for the certification exam!

**Comment by vpulyala** [comment]
Congratulations to the winners 👍

**Comment by mvpkenlin** [comment]
oh, i am not in this luck, cannot see my name. 😭

👍Congratulations to the 11 x winners 👍

**Comment by jccampanero** [comment]
@akash1134, @queenofcorgis Lili, I apologize for the late reply.

Thank you so much! Wow, I honestly did not expect to be one of the voucher winners. It's fantastic!

I've mentioned this many times, but every single time it was true: thank you very much for this amazing learning experience, for the course materials, for your support, for your kindness. You've done a great job. I'm really excited to see what new content you're going to roll out next!

I also want to express my gratitude to my coursemates. Their enthusiasm has been inspiring to see.

Congratulations to the rest of the winners! I wish you all the best with the exam.

And finally, thank you very much for the actual GitHub Copilot product! It's a wonderful piece of software!

**Comment by Akash1134** [comment]
@geoffreywiseman Please note that we're awaiting your details via the [form submission](https://forms.gle/2fJ2VVdNWnaSxNs87). You have plenty of days to so :) Last date for filling the form is December 31, 2024, 10 AM EST. Once we have your details, our team will send out the vouchers.

**Comment by geoffreywiseman** [comment]
Wow -- am I the last to fill in the form?

Congratulations to all the other winners, and I hope that everyone else enjoyed the experience and learned a lot.  I'll go fill in the form.

**Reply by Has1224** [reply]
Great

**Comment by thadavarthi** [comment]
Just now I went and filled the form.

**Reply by Has1224** [reply]
Nice

**Comment by Akash1134** [comment]
### Important Update ℹ️ 

We have all the details we needed from the winners. As our teams are currently on winter break ☃️  , there will be a slight delay in distributing vouchers. You can expect to receive them by the second week of January. We appreciate your understanding and patience.

**Reply by jccampanero** [reply]
Thank you for the information @Akash1134. Please, do not worry, take the time you need to.

Thank you very much again!

**Reply by Ronaldo-001** [reply]
No Problem.  Take your time!

**Reply by ruslan-intech** [reply]
No problem. Merry Christmas and Happy New Year!

**Reply by geoffreywiseman** [reply]
Thanks for letting us know -- happy holidays to all. ❄️🌲🎊

**Comment by Akash1134** [comment]
The certification vouchers have been sent to your email. 🎉 

We wish you the very best for your Copilot certification exam.  Feel free to share your exam experience with us—we’d love to hear from you!

Thank you!

**Comment by geoffreywiseman** [comment]
Just finished my exam, passed (91%)!  Looking forward to getting my detailed report and figuring out where the gaps were and checking what I missed. @Akash1134 Is it ok to share the parts that I found the most challenging so people can study those areas more?  (Not the specific questions, obviously, just the topics that I felt fuzzy on during the exam).

All in all, the process was fine.  Some links in one of the emails didn't work well for me, but I was able to work around that. online certifications always feel a bit ... weird, with all the camera stuff and showing your workspace, but that's not unique to GitHub Certifications, same was true for my AWS certification.  But then, it's nice not to travel to an exam centre too, so .... you can choose which path feel right for you. ;)

**Reply by Akash1134** [reply]
That is so amazing ✨  @geoffreywiseman Congratulations! and thanks so much for sharing this. 

I have shared your feedback with the GitHub Certifications team and will get back to you (on this thread) with an update.

**Reply by Akash1134** [reply]
Hi @geoffreywiseman We’re absolutely fine with you sharing your exam experience, especially around the topic areas from the domains that you found particularly challenging or think deserve extra study focus. However, please be mindful not to share any specific questions or answers from the exam.

We’d love to hear more! If there is any part of your feedback that you want to share with us directly, feel free to share anytime at [gh-certification@github.com](mailto:gh-certification@github.com).

Thanks so much!

**Reply by geoffreywiseman** [reply]
## Email Links
These links from the "Your GitHub Copilot Test with PSI is Coming Up" email didn't work:
```
<li>Review the <a href=3D"examregistration.github.c=
om">examregistration.github.com</a> site for updated information.</li>
                        <li>You will need to log into <a href=3D"examregist=
ration.github.com">examregistration.github.com</a> and click on the link in=
 the "appointments" section to launch your exam.</li>
```
The view raw source of this message looks a little odd in places, as you can see, but if I tried clicking the links, nothing happened. It looks like the scheme of the URL is missing, but ... I haven't looked into it in detail.

I had some time before the exam started, and was able to get that sorted before the exam time, but it's probably worth checking into.

## Percentage
After the exam and in the email, the score was listed as 91%, but one of the web portals I logged into lists it as 92%. Would be good if those numbers were consistent. ;)

## Challenging Subjects
The areas where I didn't get 100% were:
- GitHub Copilot plans and features
- How GitHub Copilot works and handles data
- Privacy fundamentals and content exclusions

In particular, I felt unprepared for some of the content exclusion questions, I feel like maybe I missed something in the learning path that dealt with that in more detail, or perhaps wasn't covered as well in the learning path as it could have been. Anyway, I recommend reading up on content exclusions so that you can be more confident than I was.

Anytime GitHub Copilot and Azure came up together, I felt like I hadn't read enough (or really anything) about that.  It wasn't a big part of the exam (as you can see from the exam guide), but maybe not a bad idea to read a little about the combination of the two if you haven't already.

I do wish I could see more detail about what I got wrong so that I could re-read that, but ... that would probably make it too easy for people to share information about their exams, so that's probably deliberate.

**Reply by Akash1134** [reply]
Thanks a lot for sharing such insightful feedback. ❤️

**Comment by Salgado2004** [comment]
Just passed my exam! Once again I'd like to thank the maintainers of the [GitHub Copilot Exam Prep](https://github.com/orgs/community/discussions/144443), @Akash1134 and @queenofcorgis, for the excellent work! 👏👏
For my insights, I agree with @geoffreywiseman that some topics could be covered a little bit more in detail, since some questions are tricky and I personally fell for some of them 😅 But in general I believe the content available in these threads is well aligned with the exam and its questions.

**Reply by Akash1134** [reply]
🎉 💹  Congratulations! @Salgado2004 Thanks for sharing this incredible win!

The Copilot certification is still relatively new, and the feedback from incredible community members like you is invaluable. We’ll take the insights from this campaign and apply them to improve future iterations.

**Comment by Akash1134** [comment]
#### We’re Back! 🚀 [Another Copilot Journey Begins on February 10th! ](https://github.com/orgs/community/discussions/150617)


Hey everyone! 👋

Building on the incredible momentum of this campaign, we’re thrilled to [bring it back](https://github.com/orgs/community/discussions/150617)—this time, even more practical with [Copilot Free](https://github.blog/news-insights/product-news/github-copilot-in-vscode-free/)! 🎉

We absolutely loved the engagement, insights, and inputs you shared throughout the campaign, and we’re leveling things up with:

✅ Enhanced resources to help you prepare with confidence
✅ Practice questions more closely aligned with the actual exam (Yes! we've implemented your feedback)
✅ Certification vouchers to validate your Copilot and AI skills

One thing remains the same—our goal is to help you master Copilot while learning together as a community!

📅 Refer your friends to [Join us! starting February 10th](https://github.com/orgs/community/discussions/150617) for another 4-week learning journey filled with hands-on learning with Copilot Free, challenges, and expert insights.

**Comment by AlcantraNetoM** [comment]
Hey guys  i need some help i have fail to been Approved  in the student pack 2 times, cuz of the two-factor authentication, now ive been approved but still haven got access to github copilot, it says: 

"Your Student Developer Pack will be ready soon!
We will notify you via email when your Experiences and offers become
available to you"


its been 2 days now and i still didnt get any email, and im worry that might be a problem.

**Comment by jccampanero** [comment]
I was able to pass my exam a couple of weeks ago too - sorry, I apologize for the late reply.

In the long term, I believe the course contents have been incredibly helpful in understanding the intricacies of GitHub Copilot. They have also been instrumental in enhancing the daily use of the tool.

In the short term, they enabled us to pass the certification exam. The exam was very interesting and, at times, challenging, especially with certain topics. However, it was manageable if you followed the training materials.

I do not want to miss the opportunity to say thank you again to GitHub in general, and to @Akash1134 and @queenofcorgis in particular, for the preparation of the course materials and for this great learning opportunity.

I wish you the best in all your new learning initiatives.

Thank you very much!

**Reply by ghost** [reply]
Could you please share the learning resource link you followed?

**Comment by ButeoButeo** [comment]
🚀 Exciting News! 🚀

I'm thrilled to share that I’ve officially earned the GitHub Copilot Certification from GitHub! 🎉 I'm so happy 🥳

Huge thanks to you all and special to @Akash1134 and @queenofcorgis for all the support. All the best! 👏🏻

**Reply by Akash1134** [reply]
Superb! Congratulations @ButeoButeo 👏

**Comment by johnnieng** [comment]
I’m thrilled to share that I’ve successfully passed the GitHub Copilot Certification! 🎉 This journey deepened my understanding of how Copilot’s AI-powered code completion can streamline development workflows. From writing boilerplate code to tackling complex logic, Copilot’s contextual suggestions are a game-changer for productivity.

A huge thanks to the GitHub community for the amazing resources and support! I’m excited to leverage these skills in my projects and contribute even more to open-source. If anyone’s curious about the certification process or tips, feel free to ask! 🚀

Special thanks to @Akash1134 and @queenofcorgis for all the support!👏🏻

#GitHubCopilot #Certification #Coding

![image](https://github.com/user-attachments/assets/e8920f1d-2ad2-443b-9a22-3a8ad200bc67)

**Reply by Akash1134** [reply]
Amazing @johnnieng We're proud of your achievement. Congratulations! 🎉 Thanks for sharing 🙂
